packages/fxa-graphql-api/src/backend/legal.service.ts

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ConfigService } from '@nestjs/config';
+
+import { AppConfig } from '../config';
+import { determineLocale } from 'fxa-shared/l10n/determineLocale';
+import { Injectable } from '@nestjs/common';
+import fetch from 'node-fetch';
```

Comment:
Once we migrate to Node 18, we can probably switch to the built-in [`fetch`](https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#fetch) API. 🎉 

```sh
git grep -En '"(axios|node-fetch|got)"' packages/*/package.json | cat

# packages/fxa-auth-client/package.json:30:    "node-fetch": "^2.6.7"
# packages/fxa-content-server/package.json:164:    "got": "11.8.5",
# packages/fxa-event-broker/package.json:53:    "axios": "^0.26.0",
# packages/fxa-payments-server/package.json:147:    "node-fetch": "^2.6.7",
```

Also mildly curious that `node-fetch` doesn't seem to be in the [packages/fxa-graphql-api/package.json](https://github.com/mozilla/fxa/blob/FXA-6809/packages/fxa-graphql-api/package.json) file.

Reply:
> Once we migrate to Node 18,

We actually landed the PR for this yesterday 👀

Reply:
Hmm, it'd be better not to use. I'll update the code.

Reply:
Meh, mocking is weird. Leaving as is for now. Sure we a have a few of these to clean up any ways. Also I noticed that fetch is technically still [experimental](https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#fetch) in v18. 🤷 .